core/src/test/scripts/test_data/study_es_0/data_gene_panel_matrix.txt

```diff
@@ -0,0 +1,2 @@
+SAMPLE_ID	study_es_0_mutations
+TCGA-A1-A0SB-01	example_gene_panel_stable_id
```

Comment:
In addition to a matrix file, I think it could be useful to add a `gene_panel` field to in one of the other `meta_` files in this test data set. The paragraph in `docs/File-Formats.md` after the one that explains gene panel matrix files seems to suggests that it's supported as an optional field in most data types.